apps/mobile-wallet/src/components/footers/FooterMenuItem.tsx

```diff
@@ -61,10 +61,7 @@ const FooterMenuItem = ({ options, isFocused, routeName, target, navigation }: F
 
   return (
     <TouchableWithoutFeedback onPress={onPress} key={label}>
-      <Tab active={isFocused}>
-        {Icon}
-        <TabText isActive={isFocused}>{label}</TabText>
```

Comment:
nit: Is there a point in keeping the `label` var if it's not displayed and only used as a `key`? I think I'd delete it and simply use `routeName` as the `key`.

Reply:
Good point, fixed.